docs/extend/extend-apiml/onboard-wizard.md

```diff
@@ -24,3 +24,8 @@ The following procedure describes how to onboard your REST service with the Wiza
    
 If you see your service in the list of API Catalog available services, you have onboarded your service successfully.
 
+### Prefilling fields
+
+After opening the wizard, you may upload a complete or partial YAML file to prefill the fields. The YAML file will be validated and the form fields will be populated.
+
+![YAML Prefill](../../images/api-mediation/wizard-yaml-prefill.png)  
```

Comment:
I'm not clear how the option "Or fill the fields:" works. Do we need to provide any instructions to the user about which fields to fill and where, or is this clearly described in the Wizard?

Reply:
That option just invites the user fill the form as they would normally, as in the screenshot attached. I don't think more instructions are necessary, but I would agree that the "Or fill the fields:" option looks a bit out of context in the image in the PR. So, I updated the image to exclude it.

![image](https://user-images.githubusercontent.com/17074399/162646179-93df8445-6c6c-46f7-b969-2cc389bd5e58.png)

Reply:
If I understand correctly, filling the fiends IS an option, but the feature you are trying to highlight is using the "Choose file" option, is that right? If so, it seems that the updated screenshot will suffice, but only if you don't want to bring the "fill the fields" option to the reader's attention. 

Also, I see this step appears after the validation step. Shouldn't this be before the validation step?

Reply:
Correct, the user can either fill in the fields manually, or upload a pre-existing YAML file to populate some/all fields. So, in the screenshot we want to highlight the upload feature.

The YAML upload does happen before validation. I could put it as "Step 3 (Optional)", and move the current steps 3-6 up one number. What do you think?

Reply:
@janan07 Done